worker/games.py

```diff
@@ -137,30 +137,37 @@ def required_net(engine):
     return net
 
 
-def verify_required_cutechess(cutechess):
-    print("Obtaining version info for {} ...".format(os.path.basename(cutechess)))
-    try:
-        with subprocess.Popen(
-            [cutechess, "--version"],
-            stdout=subprocess.PIPE,
-            universal_newlines=True,
-            bufsize=1,
-            close_fds=not IS_WINDOWS,
-        ) as p:
-            pattern = re.compile("cutechess-cli ([0-9]*).([0-9]*).([0-9]*)")
-            for line in iter(p.stdout.readline, ""):
-                m = pattern.search(line)
-                if m:
-                    print("Found: ", line.strip())
-                    major = int(m.group(1))
-                    minor = int(m.group(2))
-                    patch = int(m.group(3))
-    except (OSError, subprocess.SubprocessError) as e:
-        print("Exception running cutechess-cli:\n", e, sep="", file=sys.stderr)
-        raise FatalException("Not working cutechess-cli - sorry!")
+def verify_required_cutechess(testing_dir, cutechess):
+    print("Obtaining version info for {} ...".format(os.path.join(testing_dir,cutechess)))
+    os.chdir(testing_dir)
+    with subprocess.Popen(
+        [os.path.join(".",cutechess) + " --version"],
+        shell=True,
```

Comment:
It seems that the best practice is to use `shell=True` only when a shell feature is required
https://stackoverflow.com/questions/3172470/actual-meaning-of-shell-true-in-subprocess

Without `shell=True` the command must a list of tokens.